include/sdlmain.h

```diff
@@ -164,6 +164,12 @@ struct SDL_Block {
 		SCREEN_TYPES type = SCREEN_SURFACE;
 		SCREEN_TYPES want_type = SCREEN_SURFACE;
 	} desktop = {};
+	struct {
+		int cycles_num = 0;
```

Comment:
minor: `num_cycles`? (I'm used to seeing such var names start with `num_`)

Reply:
OK, changed.